source/sync.txt

```diff
@@ -8,21 +12,129 @@
    :titlesonly:
    :hidden:
    
-   What is {+sync+}? </sync/learn>
+   Get Started </sync/get-started>
    Configure and Update Your Data Model </sync/configure-your-data-model>
    Define Data Access Patterns </sync/data-access-patterns>
    Configure Sync </sync/configure-device-sync>
    Handle Errors </sync/error-handling>
    Add Sync to a Local-Only App </sync/local-to-sync>
+   Production Checklist </sync/production-checklist>
    Sync Production Load Testing </sync/production-load-testing>
    Event Library </sync/event-library>
+   Technical Details </sync/details>
+
+.. introduction::
+
+   You're developing a mobile app. Your users want their data saved on the cloud
+   and accessible from their other devices. Network access on a mobile device
+   can be hit or miss, so you write data locally on the device first. A
+   background process then synchronizes the data to the cloud and resolves any
+   conflicting writes.
+ 
+   {+sync+} does all that... and more!
+
+   .. button:: Start Building with a Template Starter App
+      :uri: https://www.mongodb.com/realm/register?tck=docs_CTA_app_services
+
+.. image:: /images/plp/Spot_AzureBlue_Sync_Tech_RealmSync.png
+   :alt: Atlas Device Sync Illustration
+
+.. kicker:: What It Is
+
+Key Features
+------------
+
+.. card-group::
+   :columns: 3
+   :style: compact
+
+   .. card::
+      :headline: Realm in the Front, MongoDB in the Back
+      :icon: /images/icons/General_FEATURES_IntelligentEdge3x.png
+      :icon-alt: An icon depicting a mobile phone connecting with the cloud
+
+      {+sync+} is a bridge between client apps using the :realm:`Realm SDKs </>`
+      and a MongoDB instance running in Atlas. Realm is a lightweight database
+      optimized for mobile development.
+      
+   .. card::
+      :headline: Offline First
+      :icon: /images/icons/Technical_REALM_Offline3x.png
+      :icon-alt: An icon depicting network loss
+
+      Realm Database and {+sync-short+} seamlessly handle intermittent
+      connectivity so users can continue using your app regardless of their
+      current network status.
+
+   .. card::
+      :headline: Conflict Resolution
+      :icon: /images/icons/Technical_REALM_ConflictResolution3x.png
+      :icon-alt: An icon depicting conflict resolution
+
+      {+sync-short+} handles conflicts for you, so you don't have to write
+      complex custom code to resolve conflicting writes from multiple clients.
+      Read more: :ref:`conflict-resolution`
```

Comment:
The styling on this :ref: is weird because of the card formatting. Can we change this to a CTA instead?

```suggestion
      :cta: Read more about Conflict Resolution
      :url: https://www.mongodb.com/atlas-app-services/sync/details/conflict-resolution/#std-label-conflict-resolution
      :icon: /images/icons/Technical_REALM_ConflictResolution3x.png
      :icon-alt: An icon depicting conflict resolution

      {+sync-short+} handles conflicts for you, so you don't have to write
      complex custom code to resolve conflicting writes from multiple clients.
```